aml/org.testeditor.aml.dsl.tests/src/org/testeditor/aml/dsl/tests/parser/validation/ValidationTest.xtend

```diff
@@ -53,13 +51,115 @@ class ValidationTest extends AbstractValidationTest {
 		input.parse(ValueSpace).assertError(
 			REG_EX_VALUE_SPACE,
 			REG_EX_VALUE_SPACE__EXPRESSION__INVALID,
-			'''
-				The given expression is not a valid regular expression in Java:
-				java.util.regex.PatternSyntaxException: Illegal repetition near index 0
-				${
-				^
-			'''.toString.trim
+			"The given expression is not a valid regular expression in Java:",
+			"java.util.regex.PatternSyntaxException: Illegal repetition near index 0",
+			"${",
+			"^"
 		)
 	}
 
+	@Test
+	def void testUsageOfDefaultLocatorStrategy() {
+		// given
+		val input = '''
+			package org.test
+
+			import org.testeditor.dsl.common.testing.DummyLocatorStrategy
+			import org.testeditor.dsl.common.testing.DummyFixture
+
+			interaction type click {
+				label = "Click on"
+				template = "click on" ${element}
+				method = DummyFixture.clickOn(element, locatorStrategy)
+				locatorStrategy = SINGLE
+			}
+		'''
+
+		// when
+		val amlModel = input.parse
+		amlModel.assertNoErrors
```

Comment:
If this is what you want to test it should be in the `// then` part. Otherwise this test is not a validation test.